build-prisma-fmt/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "@prisma/wasm-prisma-fmt",
+  "version": "0.1.2",
+  "description": "",
+  "main": "src/index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/tomhoule/prisma-fmt-build.git"
+  },
+  "author": "Prisma",
+  "license": "MIT",
```

Comment:
@janpio is the license correct?

Reply:
I think we should use Apache 2 https://github.com/prisma/prisma-engines/blob/master/LICENSE

Reply:
```suggestion
  "license": "Apache-2.0",
```